tests/util/file-storage.tests.js

```diff
@@ -26,24 +22,24 @@ const storage = new FileStorage(
 	'http://localhost:4569',
 	encryptionKeyId);
 
-Bluebird.promisifyAll(s3);
-
 describe('File storage utility', () => {
 
 	before(done => {
-		s3.createBucketAsync({ Bucket: testBucket })
+		s3.createBucket({ Bucket: testBucket })
```

Comment:
In hindsight, the promises actually make this uglier. :(
```suggestion
		s3.createBucket({ Bucket: testBucket }, done);
```